rails_application/db/schema.rb

```diff
@@ -60,6 +60,7 @@
     t.decimal "percentage_discount", precision: 8, scale: 2
     t.decimal "total_value", precision: 8, scale: 2
     t.decimal "discounted_value", precision: 8, scale: 2
+    t.decimal "time_promotion_discount", precision: 8, scale: 2
```

Comment:
Just a thought, what if we had jsonb column that would keep the time promotion information? It could be rendered directly in the frontend.

Reply:
Do you mean to have something like this: `{discount_value: 50, type: "time_promotion"}` ? If we would like to keep time promotions like that, then we probably should keep percentage discounts in a similar format and adjust the `percentage_discount` and `time_promotion_discount_value` columns in the `orders` table as well. 
What do you think about having another table for discounts, such as `client_orders_discounts` ?